docs/chapters/client/mocking.md

```diff
@@ -6,55 +6,36 @@ In a model-driven design context, it is therefore extremely convenient to be abl
-
-To create a fully mock context, Typetta provides a utility function, generated similarly to the ``DAOContext``, which receives exactly the same parameters as its constructor, excluding references to the data source.
-
-```typescript
-const daoContext = mockedDAOContext();
-```
-
-The ``DAOContext`` thus obtained can be used in the same way as that already described in the previous chapters, making the presence or otherwise of the data source transparent to the rest of the system. The mock data source will start empty, with no data and no structure.
-
-Note that the creation of a mock context allows the use of entities declared in the data model as ``@entity`` but without the annotation of the specific driver ``@mongodb`` or ``@sql``. This is particularly useful in contexts where you want to use Typetta even before designing the database in which you want to historicise the data.
-
-For entities declared only as ``@entity``, the DAO that is produced will appear and function like a DAO accessing MongoDB, and the same applies for entities declared as ``@entity @mongodb``. For entities declared as ``@entity @sql``, by contrast, a DAO with SQL database access capabilities will be produced.
-
-If there are entities of the type ``@entity @sql``, since access to data on relational databases requires the presence of previously created tables, it is necessary to call an additional ``createTables`` initialisation method on the mock ``DAOContext``.Below is an example:
-
-```typescript
-const daoContext = mockedDAOContext();
-daoContext.createTables();
-```
-
 ## Mocking some data sources
 
-In some cases, it may be useful to create a ``DAOContext`` that has some real and some mock data sources.
+In some cases, it may be useful to create a ``DAOContext`` that has some real and some mock data sources. Every entity associated with a data source instantied with the literal ``'mock'`` will be mocked.
 
-The ``mockedDAOContext`` feature allows you to explicitly specify all the data sources foreseen by the data model, exactly as you can do by instantiating the ``DAOContext`` directly. Data sources that are not specified are automatically mocked by the system.
-
-Below is an example in which a default MongoDB database is configured; each additional datasource is automatically mocked by Typetta.
+Below is an example in which a default mongodb database is mocked; the other data source instead is a real mongodb instance.
 
 ```typescript
 const mongoClient = new MongoClient(process.env.MONGODB_URL!);
 const mongoDb = mongoClient.db(process.env.MONGODB_DATABASE_NAME);
 
-const daoContext = mockedDAOContext({
+const daoContext = new DAOContext({
   mongodb: {
-    default: mongoDb,
-    // another not specified data source => mocked
+    default: 'mock',
+    other: mongoDb
   }
 });
 ```
 
+All the access to entities belonging to the default mongodb data source will be redirected to an in-memory implementation and will not use any mongodb database.
```

Comment:
"All the access to" => "Every operation on"
mongodb => MongoDB

Reply:
Aggiungerei:
"You can easily mock your entire data access layer mocking every data source. In this case you don't need a real database at all."